README.Rmd

```diff
@@ -14,7 +14,6 @@ editor_options:
 <!-- badges: start -->
 [![R-CMD-check](https://github.com/frankiecho/robust.prioritizr/actions/workflows/R-CMD-check.yaml/badge.svg)](https://github.com/frankiecho/robust.prioritizr/actions/workflows/R-CMD-check.yaml)
 [![Codecov test coverage](https://codecov.io/gh/frankiecho/robust.prioritizr/graph/badge.svg)](https://app.codecov.io/gh/frankiecho/robust.prioritizr)
-[![CRAN status](https://www.r-pkg.org/badges/version/robust.prioritizr)](https://CRAN.R-project.org/package=robust.prioritizr)
```

Comment:
I would suggest keeping the CRAN status badge. This will help tell people that the package is not currently on CRAN, and this badge will automatically update when the package is on CRAN.